gradle/run.gradle

```diff
@@ -39,6 +39,7 @@ testClusters {
     testDistribution = 'archive'
     if (numZones > 1) numberOfZones = numZones
     if (numNodes > 1) numberOfNodes = numNodes
+    systemProperty 'opensearch.experimental.feature.extensions.enabled', 'true'
```

Comment:
I think this slipped in accidentally @scrawfor99

Reply:
Shoot my bad.